controllers/apply/reaching-communities/processor.test.js

```diff
@@ -9,14 +9,23 @@ const jsdom = require('jsdom');
 const { JSDOM } = jsdom;
```

Comment:
The set up for this test was un-idiomatic and the new version of jest flagged this as an error, these changes fix the setup code to run per test rather than inline in the `describe` block.